WarmUp/WarmUp.csproj

```diff
@@ -5,4 +5,8 @@
     <TargetFramework>netcoreapp2.2</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\GenderPayGap.Extensions\GenderPayGap.Extensions.csproj" />
+  </ItemGroup>
+

```

Comment:
Similarly to my other comment, I think WarmUp shouldn't depend on the other projects.

Reply:
Fixed